SCClassLibrary/Common/Quarks/Quarks.sc

```diff
@@ -384,7 +385,11 @@ Quarks {
 		});
 	}
 	*isPath { |string|
-		^string.findRegexp(regex.isPath).size != 0
+		if(thisProcess.platform.name !== 'windows', {
+			^string.findRegexp(regex.isPath).size != 0
+		}, {
+			^(string.findRegexp(regex.isPath).size != 0).and(string.findRegexp(regex.isURL).size == 0)
+		});
```

Comment:
r = "\\\\|/"
First it's not from me, but I think it's correct, though it's likely over-general.
You get 4 `"\"` because it looks for an already escaped `"\"`. So it looks for `"\\"` and escapes each of them in the match string
`"/"` has to be there because that's also allowed in Windows. It would be nice to leave it away, as it would make distinguishing URLs a lot easier, but you wouldn't want to tell people "don't use forward slashes" if it's actually nicer to do so (no escaping needed, cross-platform...). I guess quite a bit of code depends on that "/" can be interpreted as path delimiter in Windows.